src/utils/index.js

```diff
@@ -634,3 +634,25 @@ export function getStakedAmountUSD(campaign, nativeCurrencyUSDPrice, nativeCurre
     ).toString(),
   );
 }
+
+/**
+ * Given a URI that may be ipfs, ipns, http, or https protocol, return the fetch-able http(s) URLs for the same content
+ * @param uri to convert to fetch-able http url
+ */
+export function uriToHttp(uri) {
+  const protocol = uri.split(':')[0].toLowerCase();
+  switch (protocol) {
+    case 'https':
+      return [uri];
+    case 'http':
+      return ['https' + uri.substr(4), uri];
+    case 'ipfs':
+      const hash = uri.match(/^ipfs:(\/\/)?(.*)$/i)?.[2];
+      return [`https://ipfs.io/ipfs/${hash}/`, `https://ipfs.io/ipfs/${hash}/`];
```

Comment:
why are you returning the same `https://ipfs.io/ipfs/${hash}/` twice?

Reply:
Actually I think I can remove this function, since all the URLs in the list are http

Reply:
Dind't notice that, that should be http